lib/utils.js

```diff
@@ -22,11 +22,26 @@ function isDomElement(o) {
   return o instanceof HTMLElement || o && o.nodeType > 0;
 }
 
-function renderTemplate({template, templateHelpers, data}) {
+function renderTemplate({template, defaultTemplate, config, data}) {
   var hogan = require('hogan.js');
   var forEach = require('lodash/collection/forEach');
+  var options = config.options;
   var content;
 
+  if (template === null || typeof template === 'undefined') {
```

Comment:
I get the edge case here (people redefining `undefined` global var). But those websites have other issues that we do not want to fix here.